urbansearch/clustering/preprocessing.py

```diff
@@ -0,0 +1,40 @@
+#Tokenizes, Stems, removes stop words and counts
+
+import re
+
+from nltk.tokenize import word_tokenize
+from nltk.stem.snowball import SnowballStemmer
+from nltk.corpus import stopwords
+from collections import Counter
+
+class PreProcessor:
+    def __init__(self, lang='dutch'):
```

Comment:
Please move this to a file `config.py` in the root directory; we keep our settings in that file.